uber/site_sections/preregistration.py

```diff
@@ -16,9 +16,7 @@ def check_prereg_reqs(attendee):
 
 
 def check_dealer(group):
-    if not group.address:
-        return 'Dealers are required to provide an address for tax purposes'
-    elif not group.wares:
+    if not group.wares:
         return 'You must provide a detailed explanation of what you sell for us to evaluate your submission'
```

Comment:
any reason for removing asking them for address? we do actually need that info to send to the MD comptroller later.  or does this not have the UI in place to submit it yet?

also not worth holding up this PR, just keep in mind we do need to collect their address, either when they apply, or when they're approved.